app/components/StartServer/StartServer.css

```diff
@@ -5,7 +5,7 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    background: linear-gradient(180deg, #f0f0f0, #e0e0e0);
+    background: linear-gradient(180deg, #f9f9f9, #f2f2f2, 5%, #f2f2f2, 95%, #d9d9d9);
```

Comment:
yes, I was playing around with where to put this and must have left it in a random place